app/frontend/components/domains/home/review-manager/configuration-management-screen/index.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Container,
+  FormControl,
+  FormLabel,
+  Grid,
+  GridItem,
+  HStack,
+  Heading,
+  Input,
+  Text,
+  VStack,
+} from "@chakra-ui/react"
+import { Info } from "@phosphor-icons/react"
+import { t } from "i18next"
+import { observer } from "mobx-react-lite"
+import React from "react"
+import { useParams } from "react-router-dom"
+import { useMst } from "../../../../../setup/root"
+import { HomeScreenBox } from "../../home-screen-box"
+import { SectionBox } from "../../section-box"
+
+export const ConfigurationManagementScreen = observer(function ConfigurationManagementScreen() {
+  const i18nPrefix = "home.configurationManagement"
+  const { jurisdictionId } = useParams()
+  const {
+    jurisdictionStore: { getJurisdictionById },
+  } = useMst()
+  const jurisdiction = getJurisdictionById(jurisdictionId)
```

Comment:
using the useJurisdiction hook will make the breadcrumbs and other such things work properly. I will add a commit for this.

Reply:
also it lets us default to loading and error screens 👯